tests/unit/core/test_api_errors.py

```
@@ -0,0 +1,355 @@
+    "(34.118.226.109), port 5432 failed; "
+    "postgresql://db-user:db-password@pgbouncer-rw.internal/app"
+)
+
+
+def _build_app() -> FastAPI:
+    app = FastAPI()
+    register_api_error_handlers(app)
+    app.add_middleware(ServerErrorSanitizationMiddleware)
+
+    @app.get("/unhandled")
+    async def unhandled():
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `unhandled`

Add return type annotation: `NoReturn`

(ANN202)